201-aci-linuxcontainer-volume-emptydir/azuredeploy.parameters.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
since you have defaultValues for these, just empty the parameters object...

Reply:
I tried to delete defaultValues of these parameters instead.

Reply:
No, we want defaultValues in the template whenever possible - leave the param file for GEN* values.  Makes it easier to deploy when the template has defaultValues